src/components/typography/label-large.tsx

```diff
@@ -1,10 +1,13 @@
+import { TypographyProps } from 'boilerplate-react-native/src/types/typography';
+import { Text } from 'native-base';
```

Comment:
Would like to avoid taking a dependency on `native-base`

Reply:
I am ok to use theme from native base, but would like our components to be independent of it, is that possible?